ravencoin/types.go

```diff
@@ -79,11 +79,12 @@ const (
 // Fee estimate constants
 // Source: https://bitcoinops.org/en/tools/calc-size/
 const (
-	MinFeeRate            = float64(0.00001) // nolint:gomnd
-	TransactionOverhead   = 12               // 4 version, 2 segwit flag, 1 vin, 1 vout, 4 lock time
-	InputSize             = 68               // 4 prev index, 32 prev hash, 4 sequence, 1 script size, ~27 script witness
-	OutputOverhead        = 9                // 8 value, 1 script size
-	P2PKHScriptPubkeySize = 25               // P2PKH size
+	MinFeeRate                  = float64(0.015) // nolint:gomnd
+	TransactionOverhead         = 12             // 4 version, 2 segwit flag, 1 vin, 1 vout, 4 lock time
+	InputSize                   = 68             // 4 prev index, 32 prev hash, 4 sequence, 1 script size, ~27 script witness
```

Comment:
The requirement of MinFeeRate being 0.015 is still not fully understood.

Reply:
Fixed! 👍